cloudbuild.yaml

```
@@ -20,7 +20,7 @@ steps:
     args:
       - "-c"
       - |
-        gsutil cp /mnt/data/blaise-publish-bucket-metadata.zip gs://ons-blaise-$_ENV-blaise-cloud-functions-source/
+        gsutil cp /mnt/data/blaise-publish-bucket-metadata.zip gs://ons-blaise-v2-$_ENV-blaise-cloud-functions-source/
```

Minor thing but would this be cleaner as project ID
```suggestion
        gsutil cp /mnt/data/blaise-publish-bucket-metadata.zip gs://$PROJECT_ID-blaise-cloud-functions-source/
```

---

Agreed and modified :)